### PR TITLE
enable test for Qt6

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -368,10 +385,10 @@ jobs:
           TEST_BATCH: ${{matrix.test-batch}}
           QGIS_HTTPBIN_HOST: "localhost:8082"
         run: |
-          DOCKERFILE=$( ( [[ ${TEST_BATCH} == "ORACLE"   ]] && echo "docker-compose-testing-oracle.yml" ) \
-                     || ( [[ ${TEST_BATCH} == "POSTGRES" ]] && echo "docker-compose-testing-postgres.yml" ) \
+          DOCKERFILE=$( ( [[ ${{ matrix.test-batch }} == "ORACLE"   ]] && echo "docker-compose-testing-oracle.yml" ) \
+                     || ( [[ ${{ matrix.test-batch }} == "POSTGRES" ]] && echo "docker-compose-testing-postgres.yml" ) \
                      || echo "docker-compose-testing.yml" )
-          [[ ${TEST_BATCH} == "ORACLE" ]] && sudo rm -rf /usr/share/dotnet/sdk
+          [[ ${{ matrix.test-batch }} == "ORACLE" ]] && sudo rm -rf /usr/share/dotnet/sdk
```
Comment (MorriganR) [inline .github/workflows/run-tests.yml:391]: After https://github.com/actions/virtual-environments/issues/3338 this line doesn't make sense and can be deleted.

Comment (3nids) [reply .github/workflows/run-tests.yml:391]: can you maybe make a separate PR with this?
I doubt this PR will be merged anytime soon...

**Comment by 3nids** [comment]
tests are now running and all failing:
`/root/QGIS/build/output/bin/test_app_fieldcalculator: error while loading shared libraries: libqwt-qt5.so.6: cannot open shared object file: No such file or directory`

**Comment by nirvn** [comment]
Hmm odd, qt6 core+analysis don't need qwt lib. Why would it look for that?

**Comment by 3nids** [comment]
superseded by #44395

**Pull request closed without merging**